bp_be/src/include/bp_be_ctl_defines.vh

```diff
@@ -132,6 +134,9 @@ typedef struct packed
   logic load_fault_v;
   logic store_fault_v;
   logic cache_miss_v;
+  logic itlb_fill_v;
+  
+  logic [rv64_eaddr_width_gp-1:0] pc;
```

Comment:
PC is already passed in instr_metadata, take it out of the exception pipe

Reply:
The mem_resp outputs the itlb fill results(aside from the tlb entry) in an exception struct. I added the pc section to pass the fill vaddr to FE

Reply:
You can still pass it to FE.  But it's already propagating down the pipe, we don't need to duplicate it

Reply:
Actually, we don't need to pass the vaddr in the itlb fill result struct at all, then

Reply:
Wait no, I'm mistaken.  This is the result of the fill, not the request.  But still, don't put it in the exception pipe. Directly output the fill result to the director

Reply:
The exception pipe is for exceptions that occur during the execution of an instruction. A TLB fill response is not an exception, but a request is

Reply:
So just add separate output ports for the itlb fill response in the bp_be_mem_top module?

Reply:
Yep